manager/controllers/app/fybrik_notebook_test.go

```diff
@@ -110,55 +112,89 @@ func TestS3Notebook(t *testing.T) {
+	By("Expecting write application creation to succeed")
+	g.Expect(k8sClient.Create(context.Background(), writeApplication)).Should(gomega.Succeed())
 
 	// Ensure getting cleaned up after tests finish
-	//nolint:govet
 	defer func() {
-		application := &apiv1alpha1.FybrikApplication{ObjectMeta: metav1.ObjectMeta{Namespace: applicationKey.Namespace,
-			Name: applicationKey.Name}}
-		_ = k8sClient.Get(context.Background(), applicationKey, application)
+		application := &apiv1alpha1.FybrikApplication{ObjectMeta: metav1.ObjectMeta{Namespace: writeApplicationKey.Namespace,
+			Name: writeApplicationKey.Name}}
+		_ = k8sClient.Get(context.Background(), writeApplicationKey, application)
 		_ = k8sClient.Delete(context.Background(), application)
 	}()
 
-	By("Expecting application to be created")
+	By("Expecting write application to be created")
 	g.Eventually(func() error {
-		return k8sClient.Get(context.Background(), applicationKey, application)
+		return k8sClient.Get(context.Background(), writeApplicationKey, writeApplication)
 	}, timeout, interval).Should(gomega.Succeed())
 	By("Expecting plotter to be constructed")
 	g.Eventually(func() *apiv1alpha1.ResourceReference {
-		_ = k8sClient.Get(context.Background(), applicationKey, application)
-		return application.Status.Generated
+		_ = k8sClient.Get(context.Background(), writeApplicationKey, writeApplication)
+		return writeApplication.Status.Generated
 	}, timeout, interval).ShouldNot(gomega.BeNil())
 
 	// The plotter has to be created
 	plotter := &apiv1alpha1.Plotter{}
-	plotterObjectKey := client.ObjectKey{Namespace: application.Status.Generated.Namespace, Name: application.Status.Generated.Name}
+	plotterObjectKey := client.ObjectKey{Namespace: writeApplication.Status.Generated.Namespace, Name: writeApplication.Status.Generated.Name}
 	By("Expecting plotter to be fetchable")
 	g.Eventually(func() error {
 		return k8sClient.Get(context.Background(), plotterObjectKey, plotter)
 	}, timeout, interval).Should(gomega.Succeed())
 
-	By("Expecting application to be ready")
+	By("Expecting write application to be ready")
 	g.Eventually(func() bool {
-		err = k8sClient.Get(context.Background(), applicationKey, application)
+		err = k8sClient.Get(context.Background(), writeApplicationKey, writeApplication)
 		if err != nil {
 			return false
 		}
-		return application.Status.Ready
+		return writeApplication.Status.Ready
 	}, timeout, interval).Should(gomega.Equal(true))
 
 	modulesNamespace := plotter.Spec.ModulesNamespace
 	fmt.Printf("data access module namespace notebook test: %s\n", modulesNamespace)
 
+	g.Expect(len(writeApplication.Status.AssetStates)).To(gomega.Equal(2))
+	g.Expect(writeApplication.Status.AssetStates["new-data-parquet"].CatalogedAsset).
+		ToNot(gomega.BeEmpty())
+	newCatalogedAsset := writeApplication.Status.AssetStates["new-data-parquet"].
+		CatalogedAsset
+	parts := strings.Split(newCatalogedAsset, "/")
```

Comment:
newCatalogedAsset is just assetID, not including catalog.

Reply:
Fixed. Thanks